.travis.yml

```diff
@@ -63,7 +63,7 @@ matrix:
           packages:
             - gcc-7
             - g++-7
-      env: DEVITO_ARCH=gcc-7 DEVITO_OPENMP=0 DEVITO_BACKEND=ops YC_CXX=g++-7 INSTALL_TYPE=conda RUN_EXAMPLES=False MPI_INSTALL=0
+      env: DEVITO_ARCH=gcc-7 DEVITO_OPENMP=0 DEVITO_BACKEND=ops INSTALL_TYPE=conda RUN_EXAMPLES=False MPI_INSTALL=0 OPS_COMPILER=gnu OPS_INSTALL_PATH=$HOME/build/opesci/OPS/ops
```

Comment:
how about
```
OPS_INSTALL_PATH=$HOME/OPS
```
? 

why should it go under `opesci` afterall?

Reply:
i thought that by doing that i could stick with the pattern from the section `before_scrip`
that way, the part for ops could be very similar to the part for yask
otherwise i guess i would need something like:
`cd $HOME/build/opesci/devito;`
after making for ops (see line 114)

Reply:
Fine, then it could go under $HOME/opesci , just "next to" yask ?

also, can you drop OPS_COMPILER? 'gnu' should be anyway the default choice, so it's not adding anything I'd say

Reply:
this env var is required by ops` make
it has to be set